src/mark.ts

```diff
@@ -116,7 +116,13 @@ export interface MarkDef {
   clip?: boolean;
 }
 
-export type AnyMark = CompositeMark | CompositeMarkDef | Mark | MarkDef;
+/** @hide */
+export type hiddenComposite = CompositeMark | CompositeMarkDef;
```

Comment:
Should this be HiddenComposite (Capital H?)

Reply:
Fixed